crates/matrix-sdk-sled/src/cryptostore.rs

```diff
@@ -1100,15 +1100,14 @@ impl CryptoStore for SledStore {
 
 #[cfg(test)]
 mod tests {
-    use lazy_static::lazy_static;
     use matrix_sdk_crypto::cryptostore_integration_tests;
+    use once_cell::sync::Lazy;
+
     use tempfile::{tempdir, TempDir};
 
     use super::SledStore;
-    lazy_static! {
-        /// This is an example for using doc comment attributes
-        static ref TMP_DIR: TempDir = tempdir().unwrap();
-    }
+    /// This is an example for using doc comment attributes
```

Comment:
You can remove this, it seems like somebody previous copy-pasted from an example and forgot to remove it.